vmdb/app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -477,6 +474,17 @@ def build_template_grid(templates,sort_order="DESC",sort_by="name")
     @temp[:templates] = sorted.uniq
   end
 
+  def dialog_partial_for_workflow
+    case (@edit || @options).try(:[], :wf)
+    when MiqProvisionWorkflow
+      "prov_dialog"
+    when MiqHostProvisionWorkflow
```

Comment:
there is a `try` here. So there is the question if `@edit[:wf]` is even defined.
In the case that it isn't, it used to render a dialog, now it will not.

Is that ok?